### ISSUE TITLE
Update to 10.5?

### ISSUE THREAD

**Issue by blaineam** [jboxberger/synology-gitlab#31]
Will it be possible with the postgres version to support gitlab 10.5? Will I be able to update to 10.5 from a current 10.2.5 installation? Keep up the great work.

**Comment by jboxberger** [comment]
Hi techyowl,

the postgres version can be updated to 10.5.1. I just uploaded it [here](https://github.com/jboxberger/synology-gitlab-jboxberger/releases). I think the mysql version is dead becaus Gitlab dropped the mysql support. 

Well ich have not build a migration tool from mysql to postgres sql yet. But technically you just need to migrate the database and restore the gitlab directory. 

I will give this tool a try.
https://hub.docker.com/r/sameersbn/mysql2psql/

But at the  moment i am so fully loaded with work so i have not much time for. I do my best to work out i soon.

Kind Regards

**Comment by helmut-steiner** [comment]
Hey techyowl,
For a start: 
1. Register the sameersbn/mysql2psql docker image in your synology nas docker ui. 
2. Make a new folder "mysql2psql" in the docker folder
3. Extract the two files in the attached ZIP [mysql2psql.zip](https://github.com/jboxberger/synology-gitlab/files/1797594/mysql2psql.zip). Import the json file in the docker UI to create a new container pointing to the right mysqld.sock file.  (This is not possible with the Synology Docker UI.)
4. Drop the yml file in the created mysql2psql folder and edit it to set your Maria DB  Gitlab database password.
5. Start the container, go to the container details, create a bash and type
```
cd /home
mysql2psql
```
The gitlab database should now be dumped to a postgresql file called "gitlab-dump-psql.sql" in your docker/mysql2psql folder. 

I hope this helps. 
Kind regards

**Comment by blaineam** [comment]
Hello, 
Thank you for making a tutorial on upgrading to 10.5. As of writing since I was just starting to use gitlab I didn’t have to worry too much about upgrading an existing installation as I did getting the latest working. I found a couple tutorials for synology running the latest gitlab at the time it was 10.5.1 however 10.5.2 was already released and someone on the docket-gitlab repo that the synology package uses had a pull request for a passing 10.5.2 docker build file so I built it and pushed it up to my docket hub account so I could have easy access to it. I then followed a synology guide for 10.5.1 and although it had a decent write up I got my certificates all mixed up and later found someone wrote a guide for all the correct configuration environment variables and used the synology certificate manager to mange the certificates by using a reverse proxy built into the DSM which is cool that it has that. Anyhow several days of research and attempts later I have 10.5.2 running just fine on my Synology with my repos on it and a gitlab pages and container registry enabled and working.

Keep up the awesome one click installer though because it is very very useful for people new to setting up a server. 

Respectfully,
Blaine Miller
Co-Creator of Cache & Featheread
www.blaineam.com
